From 568e0d6070021a9b805ba1fe1543e4b43a073413 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
Date: Thu, 20 Feb 2020 00:23:35 +0100
Subject: [PATCH] mk4build: also pass PKG_CONFIG_FOR_BUILD to the native
 configure

In commit 0414ad2b4e8978a14343d920a5a1f11da892eaf3, mk4build was
modified to pass a number of *_FOR_BUILD variables down to the
configure script called for building the native tools.

However, this configure script also uses the pkg-config tool, and the
pkg-config to use for the native build and the cross build may be
different, so let's also pass PKG_CONFIG_FOR_BUILD down to the
sub-configure, as PKG_CONFIG, following the same logic as the other
variables.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
Upstream: https://github.com/brltty/brltty/pull/248
[Upstream patch is different, due to other upstream changes.]
---
 mk4build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mk4build b/mk4build
index 3c64963b7..ad88ee69c 100755
--- a/mk4build
+++ b/mk4build
@@ -73,6 +73,7 @@ then
    CXXFLAGS=${CXXFLAGS_FOR_BUILD} \
    LDFLAGS=${LDFLAGS_FOR_BUILD} \
    LDLIBS=${LDLIBS_FOR_BUILD} \
+   PKG_CONFIG=${PKG_CONFIG_FOR_BUILD} \
    "${sourceRoot}/configure" \
       --disable-api \
       --disable-gpm \
-- 
2.24.1

